tg_bot.py

```diff
@@ -40,9 +41,17 @@ def display_tweet(self, context):
         print('starting display parameter')
         job = context.job
         global KILL_SIGNAL
+        previous_date = datetime.now() - timedelta(days=1)
         tid = pd.read_csv(self.dirpath + 'tid_store.csv')
-        for index, tid_store in tid.iterrows():
-            
+        tid["Date_time"]= pd.to_datetime(tid["Date_time"])
+
+        for index, tid_store in tid[::-1].iterrows():
+            scrape_date = tid_store['Date_time']
+            if scrape_date<=previous_date:
+                context.bot.send_message(job.context, text = "Rest are the tweets which were scraped before")
```

Comment:
The user of the telegram bot doesn't need to be aware of how the bot works, those are just things developer needs to be aware of. Add that as a debug or warning messages.

Reply:
okay